src/catchup/ApplyLedgerChainWork.cpp

```diff
@@ -99,7 +99,7 @@ TxSetFramePtr
 ApplyLedgerChainWork::getCurrentTxSet()
 {
     auto& lm = mApp.getLedgerManager();
-    auto seq = lm.getCurrentLedgerHeader().ledgerSeq;
+    auto seq = lm.getLastClosedLedgerNum() + 1;
```

Comment:
I think it can be good moment to rename getLastClosedLedgerNum to getLastClosedLedgerSeq

Reply:
There are many uses of `getLastClosedLedgerNum` (I'm seeing 67 on my local branch) and I think changing this would just make this PR even messier. I support the change but let's do it in a separate clean-up PR.